pages/v/[id]/index.tsx

```diff
@@ -134,4 +138,4 @@ const Playback: React.FC<Props> = ({ playbackId, shareUrl, poster }) => {
   );
 };
 
-export default Playback;
+export default Playback;
```

Comment:
Ok at the risk of sounding like a dummy, should more of this be shared between the two pages? It almost feels like this page should be pulling in what's in embed.tsx?

Reply:
Yea, I started off with that but there are a couple of departures that were too large to mix into one--

- No `<Layout />` parent for embed
- Style overrides for the Plyr to make it span the width/height 100%

I didn't want to make things difficult to maintain so I broke them out.  I can be talked out of this if you all have recommendations.

Reply:
@dylanjha what is your sentiment on this?

Reply:
There are some shared bits there, but I'm okay with the separation and a bit of in this case. If we find we have to duplicate some of this *again*, then we can extract some more pieces

Reply:
Righteous!